project.clj

```diff
@@ -4,6 +4,8 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/tools.reader "0.10.0"]
+                 [org.clojure/data.json "0.2.6"]
```

Comment:
Last I checked it was considerably faster -- but I now realize I haven't checked in a bit. Supports custom encoders, if you're in to that kind of thing.